index.js

```diff
@@ -60,7 +134,28 @@
  * };
  *
  */
+ var = allFunctions {
+ 	q0: countdown(),
+ 	q1: max(),
+ 	q2: longer(),
+ 	q3: isEven(),
+ 	q4: getAreaOfCircle(),
+ 	q5: getFullName(),
+ 	q6: reverseString(),
+ 	q7: maxArray()
```

Comment:
What we've asked you to do in this question is to store the _function itself_ in this object. What you've done is store the `return` value of each function by calling them. Instead, we should just to refer to the functions by name, rather than calling them by using parenthesis:

```js
var allFunctions = {
    q0: countdown,
    q1: max,
    q2: longer,
    ...
}
```

_Then_ you can call the functions by finding them in the object using dot notation:

```js
allFunctions.q0();
allFunctions.q1(2, 10);
...
```